### PR TITLE
New Admin Roles: Marshals, Pizza, Others

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

Adds Roles, Spawners and startingGear:
- [x] Marshal Bureau - Marshal and Deputy
- [x] Pizza Planet - Pizza Delivery Boy
- [x] We-Ya PMC - PMC Standard and Leader

Also adds:
- veteran gloves
- faction clothes
- faction headsets

Its EVENT ONLY currently.

## Media

зображення

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by Kadeo64** [comment]
do deliverymen have the ability to speak with xenos (afaik they can in cm13)

**Comment by Tunguso4ka** [comment]
> do deliverymen have the ability to speak with xenos (afaik they can in cm13)

No

File: Resources/Locale/en-US/_RMC14/ghost-roles/pmc.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+rmc-ghost-role-information-pmc-standard-name = Weyland-Yutani PMC (Standard)
+rmc-ghost-role-information-pmc-standard-description = Ensure no damage is incurred against Weston-Yamada. Make sure the Corporate Liaison is safe.
+    Deny the Weyland-Yamada's involvement and do not trust the UNMC forces.
```
Comment (TheDoctor1977) [inline Resources/Locale/en-US/_RMC14/ghost-roles/pmc.yml:3]: > the Weyland-Yamada

aggressive rebranding

Comment (Tunguso4ka) [reply Resources/Locale/en-US/_RMC14/ghost-roles/pmc.yml:3]: me forgor, also, Weston-Yamada

Comment (Tunguso4ka) [reply Resources/Locale/en-US/_RMC14/ghost-roles/pmc.yml:3]: done.

Comment (TheDoctor1977) [reply Resources/Locale/en-US/_RMC14/ghost-roles/pmc.yml:3]: you still missed the one on line 3
>  Deny the **Weyland-Yamada's** involvement and do not trust the UNMC forces.

**Comment by TheDoctor1977** [comment]
Does the CL have access to the We-Ya channel by default or will they have to get it from the PMCs in the event of their arrival?

**Comment by Tunguso4ka** [comment]
> Does the CL have access to the We-Ya channel by default or will they have to get it from the PMCs in the event of their arrival?

I planned to unbloat/fix headsets and encryption keys today/tommorow.

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**